addons/mail/models/res_config_settings.py

```diff
@@ -28,16 +28,19 @@ class ResConfigSettings(models.TransientModel):
         help='Users will still be able to render templates.\n'
         'However only Mail Template Editors will be able to create new dynamic templates or modify existing ones.')
     use_twilio_rtc_servers = fields.Boolean(
```

Comment:
If it's no longer used it could be removed (unless there is some backwards compatibility reason to keep it)